sql/migrations/1_migration_latest.sql

```diff
@@ -9414,3 +9414,92 @@ GO
 IF COL_LENGTH(N'tblEmailSettings', N'SenderDisplayName') IS NULL
 	ALTER TABLE tblEmailSettings ADD SenderDisplayName NVARCHAR(255) NULL
 GO
+
+
+DROP FUNCTION IF EXISTS [dbo].[udfGetSnapshotIndicators];
+GO
+
+CREATE FUNCTION [dbo].[udfGetSnapshotIndicators](

```

Comment:
Please create external file in store procedures folder with this.

Reply:
It's not procedure but function, I've changed definition in [sql/base/3_functions.sql](https://github.com/openimis/database_ms_sqlserver/pull/136/files#diff-867d030885cc00529d898b56f5fbac973865f5405b8787e00990823362cd9bbf). 
Should we refactor functions to have them in separate files as well?